src/index.js

```diff
@@ -142,10 +151,37 @@ function init( {
 					} else {
 						node.addEventListener( 'mouseenter', showPopup )
 					}
+
+					foundDetectLinks.push( {
+						text: node.textContent,
+						title: matches.title,
+						lang: matches.lang
+					} )
 				}
 			}
 		)
 	}
+
+	if ( debug ) {
+		/* eslint-disable no-console */
+		console.group( 'Wikipedia Preview [debugger]' )
+		console.log( `searching for the content in language "${globalLang}" within the selector of "${selector}"` )
```

Comment:
Suggestion: Searching for "${selector}" inside "${root}"... Found: ${foundSelectorLinks.length}